src/annotator/guest.js

```diff
@@ -409,6 +421,10 @@ export default class Guest {
         h._annotation = anchor.annotation;
       });
       anchor.highlights = highlights;
+
+      if (this._focusedAnnotations.has(anchor.annotation.$tag)) {
```

Comment:
The `anchor` method is quite long, which makes this change a little bit hard to evaluate in isolation. It would appear that this is additive, adding a highlight-focusing step to anchoring, where it didn't happen before, based on any "cached" focused annotation information. "Seems OK."